workbox-config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   swDest: 'public/sw.js',
   runtimeCaching: [{
     // Match any same-origin request that contains 'api'.
-    urlPattern: new RegExp('^https\:\/\/gateway.marvel.com\/v1\/public\/.+'),
+    urlPattern: new RegExp(/https\:\/\/gateway\.marvel\.com\/v1\/public\/.+/g),
```

Comment:
**Suggestion:** The RegExp constructor is not necessary when defining a regular expression. It can be simplified.
```suggestion
    urlPattern: /https\:\/\/gateway\.marvel\.com\/v1\/public\/.+/g,
```